javascript/javascript1/week2/studentmanager.js

```diff
@@ -0,0 +1,27 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (class07Students.length == 6) {
+    // Checking the array lenght
+    console.log("Cannot add more students to class 07");
+  } // checking if the element already exists in array
+  else if (class07Students.includes(studentName)) {
+    console.log(studentName + "already exists");
+  } // if the student name is "queen", push
+  else if (studentName === "Queen") {
+    class07Students.push(studentName);
+  } // checcking for empty string
+  else if (studentName == "") {
```

Comment:
I recommend to use `trim()` method on the string to accept any empty strings like `""`, `"    "`
```js
else if (studentName.trim() === "") {
```

Reply:
Done!!